src/offices/_single.html

```diff
@@ -100,22 +101,30 @@ <h2 class="u-visually-hidden">
             {% if resource.icon %}
             <div class="media_image-container
                         media_image-container__wide-margin">
-                <img class="media_image u-centered-on-mobile"
-                     width="150"
+                <img class="media_image
+                            media_image__150
+                            u-centered-on-mobile
+                            qa-resource-img"
```

Comment:
Ah yeah, I didn't think there'd be a situation where another would exist, but I can see now that's a possibility.